workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/AccountBusinessEventsRestController.java

```diff
@@ -72,7 +80,8 @@ public ResponseEntity<String> post(
 			_updateZendesk(
```

Comment:
It doesn't really make sense to pass the business events json in this method cause it can be looked up, just like you did in the scheduled job